org.eclipse.jdt.ls.target/org.eclipse.jdt.ls.tp.target

```diff
@@ -33,8 +33,9 @@
         <location includeAllPlatforms="false" includeConfigurePhase="false" includeMode="planner" includeSource="true" type="InstallableUnit">
```

Comment:
While I'm at it, to be technically correct, I should bump the 4.34-I-builds from above so that jdt.core.manipulation matches the jdt.core bundles from below. Currently the dom-with-javac branch sits on top of `I20241014-1800` .